src/test/kotlin/uk/gov/dluhc/printapi/testsupport/testdata/model/PrintResponseBuilder.kt

```diff
@@ -20,22 +20,22 @@ fun buildPrintResponses(
 
 fun buildBatchResponse(
     batchId: String = aValidBatchId(),
-    message: String = DataFaker.faker.famousLastWords().lastWords(),
+    message: String = faker.harryPotter().spell(),
```

Comment:
The tests fail sometime as famousLastWords() sometimes have special chars. hopefully harry potter spell will fix the problem. 🪄